article/helpers.py

```diff
@@ -8,26 +9,65 @@
 from . import structure
 
 
+WORDS_PER_MINUTE = 180  # Average WPM on screen
+
+
 def markdown_to_html(markdown_file_path):
     html = render_to_string(markdown_file_path)
     return markdown2.markdown(html)
 
 
+def filter_lines(lines_list):
+    """BeautifulSoup returns \n as lines as well, we filter them out.
+
+    It's a function because more filtering can be added later.
+    """
+    return filter(lambda x: x != '\n', lines_list)
+
+
+def lines_list_from_html(html):
```

Comment:
wouldn't it be better to use .md files?

Reply:
Ideally yes, but I couldn't find a library able to filter out the language markup like `###`, `-` and the link syntax or just count the content words.
The quickest way was to convert it to HTML and use `BeautifulSoup` to digest it and eliminate the blank lines.